index.html

```diff
@@ -4,9 +4,6 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  
-
-  
     <!-- google font -->
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
- [ ] `required`
- [ ] Also the speaker section is not very responsive you can see from the background color and the overlapping text.
- [ ] Please see screenshots below. 🥂 

`your template`
![image](https://user-images.githubusercontent.com/66230306/198564133-b2d4d9bb-e347-4ec8-b076-c83a0af5040b.png)

`origin design`
![image](https://user-images.githubusercontent.com/66230306/198564360-e061b76a-67dd-49a2-b5bc-f20eef667e3d.png)